src/Components/MovieCardSelected/MovieCardSelected.tsx

```diff
@@ -1,57 +1,55 @@
+import React, { useCallback, useContext, useMemo } from 'react';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faSearch } from '@fortawesome/free-solid-svg-icons';
 import { Logo } from '../Logo/Logo';
 import { transformDuration } from '../../utils/transformDuration';
 import { Movie } from '../../models/Movie';
+import { SelectedMovieContext } from '../../contexts/SelectedMovieContext';
+import { getYear } from '../../utils/getYearFromDate';
+import { joinGenres } from '../../utils/joinGenresWithComma';
 import './MovieCardSelected.scss';
 
 interface MovieCardSelectedProps {
   movie: Movie;
 }
 
-export function MovieCardSelected({ movie }: MovieCardSelectedProps) {
-  function handleGoToSearch() {
-    console.log('go to search');
-  }
-
-  function getYear(releaseDate: string): string {
-    return releaseDate.slice(0, 4);
-  }
+function MovieCardSelectedComponent({ movie }: MovieCardSelectedProps) {
+  const { title, poster_path, vote_average, genres, release_date, runtime, overview } = movie;
+  const { setSelectedMovie } = useContext(SelectedMovieContext);
```

Comment:
I would preferred to avoid dependency from context here. Especially because you use the context in parent component. So I would put the function as prop

Reply:
Done.